roles/edpm_nova/tasks/configure.yml

```diff
@@ -77,13 +78,13 @@
   check_mode: false
   delegate_to: localhost
 
-- name: Flatten configmaps into {{ edpm_nova_config_dest }}
+- name: Flatten configmaps into {{ edpm_nova_config_sync_dest }}
```

Comment:
so we could create the flattend config map in the AEE container by delegation this to the aee host and then rsyncing the result after we have merged the files.

that would avoid creating the tempory folder on the dest host it would also have the advantage of only doing the flattening once. rather then doing it for every host.

Reply:
done